src/test/regress/sql/multi_mx_function_call_delegation.sql

```diff
@@ -274,10 +274,20 @@ select mx_call_func((select x + 1 from mx_call_add(3, 4) x), 2);
 select mx_call_func(floor(random())::int, 2);
 
 -- test forms we don't distribute
-select * from mx_call_func(2, 0);
 select mx_call_func(2, 0) where mx_call_func(0, 2) = 0;
 select mx_call_func(2, 0), mx_call_func(0, 2);
 
+-- regular call in FROM can be pushed down
+select * from mx_call_func(2, 0);
+
+prepare call_func(int, int) as select $1 from mx_call_func($1, $2);
+execute call_func(2, 0);
```

Comment:
Please add a comment on why we chose to execute 6 times, newbies may not have the context around PG's magic number "5"